resources/META-INF/plugin-inspections.xml

```diff
@@ -291,6 +291,10 @@
                          shortName="DefaultValueForEnumTypeMustBeAssignable" level="ERROR"
                          implementationClass="com.intellij.idea.plugin.hybris.codeInspection.rule.typeSystem.TSDefaultValueForEnumTypeMustBeAssignable"/>
 
+        <localInspection language="XML" enabledByDefault="true" groupPath="SAP Commerce" groupName="[y] Type System"
+                         bundle="i18n.HybrisBundle" key="hybris.inspections.ts.O2MRelationCannotContainDeploymentAttribute.key"
+                         shortName="O2MRelationCannotContainDeploymentAttribute" level="ERROR"
```

Comment:
`O2MRelationCannotContainDeploymentAttribute`, it is not `attribute` rather `tag`